docs/user-guide/creating-application/overview.md

```diff
@@ -47,4 +47,57 @@ Click **Save**. The application will be moved to the selected project.
 * To remove the tags from propagation, click the symbol <img src="https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/propagate-dark.jpg" height="10"> again.
 * Click `Save`.
 
-The changes in the tags will be reflected in the `Tags` on the `Overview` section.
\ No newline at end of file
+The changes in the tags will be reflected in the `Tags` on the `Overview` section.
+
+## Configure PersistentVolumeClaim (PVC) for Build Time Optimization
+
+ A PersistentVolumeClaim (PVC) volume is a request for storage, which is used to mount a PersistentVolume (PV) into a Pod. In order to optimize build time, you can configure PVC in your application.
+
+If you want to optimize build time for the multiple target platforms (e.g., arm64, amd64), mounting PVC will provide volume directly to a pod which helps in shorter build time by storing build cache. Mounting PVC will provide storage for build cache which will not impact the normal build where the image is built on the basis of architecture and operating system of the K8s node on which CI is running.
+
+### Create PVC file
+
+* The following configuration file describes persistent volume claim e.g.,`cache-pvc.yaml`, where you have to define the metadata `name` and `storageClassname`.
+
+```bash
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cache-pvc{here comes the name of PVC}
```

Comment:
change {here comes the name of PVC} to # here comes the name of PVC
so that it goes as comment and even someone does a copy paste, it shouldn't throw any error

Reply:
Perfect!

Reply:
Done.